en/computer_vision/obstacle_avoidance.md

```diff
@@ -18,146 +28,59 @@ The desired route comes from a [ROS](http://dev.px4.io/en/ros/) node running on
 This is passed into an obstacle avoidance module (another ROS node).
 The avoidance software sends the planned path to the flight stack as a stream of `SET_POSITION_TARGET_LOCAL_NED` messages.
 
-> **Note** The only required PX4-side setup is to put PX4 into *Offboard mode*.
-  While the `SET_POSITION_TARGET_LOCAL_NED` setpoints come from a ROS collision avoidance node, to PX4 could be from any MAVLink system.
+While the `SET_POSITION_TARGET_LOCAL_NED` setpoints come from a ROS collision avoidance node, to PX4 they could be from any MAVLink system.
```

Comment:
I am not sure I understand what this means. 
Also collision avoidance is confusing since we either have collision prevention or obstacle avoidance in the PX4 jargon.

Reply:
What this was supposed to say  is that PX4 does not care about what generates the offboard messages - it just happens to be the obstacle avoidance ROS software. This isn't particularly useful to tell people, so I have deleted it.

Yes, thanks, fixed the terminology!